.github/workflows/activity.yaml

```diff
@@ -0,0 +1,29 @@
+name: Create User Activity Report
+
+on:
+  pull_request:
```

Comment:
Added this trigger for testing, we should remove it if we decide to merge the PR.

Or perhaps replace it with a `schedule:` trigger to run the job on a nightly or weekly basis?